game-server/data/handlers/quest/sanctum/_3908ToMastertheDragon.java

```diff
@@ -0,0 +1,65 @@
+package quest.sanctum;
+
+import static com.aionemu.gameserver.model.DialogAction.*;
+
+import com.aionemu.gameserver.model.gameobjects.player.Player;
+import com.aionemu.gameserver.questEngine.handlers.AbstractQuestHandler;
+import com.aionemu.gameserver.questEngine.model.QuestEnv;
+import com.aionemu.gameserver.questEngine.model.QuestState;
+import com.aionemu.gameserver.questEngine.model.QuestStatus;
+
+public class _3908ToMastertheDragon extends AbstractQuestHandler {
+
+	private final static int questId = 3908;
+
+	public _3908ToMastertheDragon() {
+		super(questId);
+	}
```

Comment:
```suggestion
	public _3908ToMastertheDragon() {
		super(3908);
	}
```
The static `questId` field here shadows the protected `questId` field from the AbstractQuestHandler, which should always be avoided.